docs/conf.py

```diff
@@ -56,11 +53,15 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
-# The short X.Y version.
-version = '4.1.0'
+
 # The full version, including alpha/beta/rc tags.
-release = '4.1.0'
+import io
+with io.open('../version.txt') as f:
+    release = f.read().strip()
```

Comment:
I did not like the `version.txt` in the first place, too. But if it is the easiest way to please RTD **and** keep the version number DRY I am fine with it.